src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/electric/ElectricCompressor.java

```diff
@@ -0,0 +1,107 @@
+package me.mrCookieSlime.Slimefun.Objects.SlimefunItem.machines.electric;
+
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.inventory.ItemStack;
+
+import me.mrCookieSlime.CSCoreLibPlugin.general.Inventory.InvUtils;
+import me.mrCookieSlime.CSCoreLibPlugin.general.Inventory.Item.CustomItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Lists.SlimefunItems;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.abstractItems.AContainer;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.abstractItems.MachineHelper;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.abstractItems.MachineRecipe;
+import me.mrCookieSlime.Slimefun.Setup.SlimefunManager;
+import me.mrCookieSlime.Slimefun.api.BlockStorage;
+import me.mrCookieSlime.Slimefun.api.energy.ChargableBlock;
+import me.mrCookieSlime.Slimefun.api.inventory.BlockMenu;
+
+public abstract class ElectricCompressor extends AContainer {
+
+    public ElectricCompressor(Category category, ItemStack item, String id, RecipeType recipeType, ItemStack[] recipe) {
+        super(category, item, id, recipeType, recipe);
+    }
+
+    @Override
+    public String getInventoryTitle() {
+        return "&9Electric Compressor";
+    }
+
+    @Override
+    public ItemStack getProgressBar() {
+        return new ItemStack(Material.TURTLE_HELMET);
```

Comment:
Just curious: Why a Turtle Helmet?

Reply:
Because couldn'r think of any item and thought why not?

Reply:
Suggestions are open for this.